obstore/python/obstore/fsspec.py

```diff
@@ -104,6 +104,12 @@ def _split_path(self, path: str) -> Tuple[str, str]:
             # no bucket name in path
             return "", path
 
+        if path.startswith(self.protocol + "://"):
+            path = path[len(self.protocol) + 3 :]
+        elif path.startswith(self.protocol + "::"):
```

Comment:
Can you give an example of where `::` would show up? Or maybe even add it to a test?